nsv13/code/modules/munitions/ammunition/torpedos/torpedo_types.dm

```diff
@@ -5,9 +5,9 @@
 	desc = "A fairly standard torpedo which is designed to cause massive structural damage to a target. It is fitted with a basic homing mechanism to ensure it always hits the mark."
 	anchored = TRUE
 	density = TRUE
-	projectile_type = /obj/item/projectile/bullet/torpedo //What torpedo type we fire
+	projectile_type = /obj/item/projectile/missile/torpedo //What torpedo type we fire
 	pixel_x = -17
-	var/speed = 1 //Placeholder, allows upgrading speed with better propulsion
+//	var/speed = 1 //Placeholder, allows upgrading speed with better propulsion
```

Comment:
If this code isn't being used we can go ahead and remove it

Reply:
Most commented stuff is yet to be removed, or there because I'm considering a different option and will be removed later.